inference-engine/tests/functional/plugin/myriad/single_layer_tests/out_shape_of_reshape.cpp

```diff
@@ -54,8 +54,8 @@ class OutShapeOfReshapeLayerTest : public testing::WithParamInterface<OutShapeOf
 protected:
     void SetUp() override {
         SetRefMode(LayerTestsUtils::RefMode::INTERPRETER);
-        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
-        configuration[VPU_CONFIG_KEY(DISABLE_REORDER)] = CONFIG_VALUE(YES);
+        configuration[InferenceEngine::MYRIAD_DETECT_NETWORK_BATCH] = CONFIG_VALUE(NO);
+        configuration[InferenceEngine::MYRIAD_DISABLE_REORDER] = CONFIG_VALUE(YES);
```

Comment:
Consider to remove this option if it's confirmed we don't support MY2 in current release. Or at least deprecte it.

Reply:
`MYRIAD_DISABLE_REORDER` is a private option. I think we will be able to remove it freely, when it will be not needed.
Marking it as deprecated will entail adding a large amount of `IE_SUPPRESS_DEPRECATED_START`/`IE_SUPPRESS_DEPRECATED_END`